### PR TITLE
Redirect to profile page if registered before.

### PR DESCRIPTION
**Pull request by apluscs**

### CODE REVIEW

**Review by GrantComm — APPROVED** [review]

**Review by meagancheese — APPROVED** [review]
LGTM

**Comment by apluscs** [comment]
Gonna wait on PR 104

File: src/main/webapp/register.js [hunk 1]
```diff
@@ -14,9 +14,23 @@
 
 function onRegisterLoad() {
   supplyLogoutLink();
+  checkRegisteredBefore();
   autofillEmail().then(prepareFormValidation());
 }
 
+// If they have registered before, redirect them to profile page.
```
Comment (jalexanderqed) [inline src/main/webapp/register.js:22]: This seems to check whether the user is currently logged in, not whether they have previously registered? If so, please change the comment and function name.

**Review by jalexanderqed — CHANGES_REQUESTED** [review]

**Comment by apluscs** [comment]
Gonna wait on PR #118

File: src/main/webapp/register.js (hunk 1 above)
Comment (apluscs) [reply src/main/webapp/register.js:22]: No it checks whether they have previously registered. Added more docu.

**Review by jalexanderqed — APPROVED** [review]

**Pull request merged**